### PR TITLE
REPLAT-2285/Use ART for SVGs

### PR DESCRIPTION
**Pull request by fourlastor**
This PR introduces an ART implementation for SVGs, also switches to pre-rendered images for the watermark svg on mobile devices.

This is due to a missing feature in ART (the `fillType` property can't be set to `evenodd`)

Web falls back to `svgs`.

There are some packages that aren't still using `@times-components/svgs`, the plan is to move them once this is stable: `brightcove-video`, `jest-serializer`, `pagination`, `video`, `watermark`.

### CODE REVIEW

File: packages/svgs/src/g.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { ART } from "react-native";
+import PropTypes from "prop-types";
+
+const { Group } = ART;
+
+const G = ({ fill, stroke, strokeWidth, opacity, children }) => {
+  const childrenWithProps = React.Children.map(children, child => {
```
Comment (craigbilner) [inline packages/svgs/src/g.js]: could split this out to make more readable

File: packages/svgs/src/path.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  d: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(ARTPath)])
+    .isRequired,
+  fill: PropTypes.string,
+  stroke: PropTypes.string,
+  strokeWidth: PropTypes.string,
+  opacity: PropTypes.string
+};
+
+Path.defaultProps = {
+  fill: undefined,
```
Comment (craigbilner) [inline packages/svgs/src/path.js]: I don't think humans should use `undefined`

File: packages/svgs/src/svg.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import * as React from "react";
+import { ART } from "react-native";
+import PropTypes from "prop-types";
+
+const { Surface, Group, Transform } = ART;
+
+const Svg = ({ width, viewBox, children }) => {
+  const [shapeX, shapeY, shapeWidth, shapeHeight] = viewBox
```
Comment (craigbilner) [inline packages/svgs/src/svg.js]: perhaps need some more robust error/edge case handling here?

Comment (fourlastor) [reply packages/svgs/src/svg.js]: would a regex work? also should we do error handling here or in proptypes?

File: packages/svgs/src/path.js (hunk 2 above)
Comment (fourlastor) [reply packages/svgs/src/path.js]: 66882ea

File: packages/svgs/src/svg.js (hunk 3 above)
Comment (fourlastor) [reply packages/svgs/src/svg.js]: 1b8fcb6

File: packages/svgs/src/g.js (hunk 1 above)
Comment (fourlastor) [reply packages/svgs/src/g.js]: c208d7d

**Pull request merged**